tests/topotests/lib/bgp.py

```diff
@@ -21,6 +21,7 @@
 from copy import deepcopy
 from time import sleep
 import traceback
+import ipaddr
```

Comment:
Please do not add `ipaddr` lib anymore. Use the `ipaddress` lib which replaced it (and is already imported at next line)